lib/actions/CodeDeployLambda.js

```diff
@@ -134,7 +138,8 @@ module.exports   = function(S) {
           // Compress
           _this.zipBuffer = zip.generate({
             type: 'nodebuffer',
-            compression: 'DEFLATE'
+            compression: 'DEFLATE',
+            platform: 'UNIX'
```

Comment:
from https://stuk.github.io/jszip/documentation/api_jszip/generate.html 
> If you set the platform value on nodejs, be sure to use process.platform. fs.stats returns a non executable mode for folders on windows, if you force the platform to UNIX the generated zip file will have a strange behavior on UNIX platforms.

@nicka, could you please, update the line to use `process.platform`?